pkg/injector/components/components.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package components
+
+import (
+	"strings"
+
+	"github.com/dapr/dapr/pkg/components/pluggable"
+
+	"github.com/dapr/dapr/pkg/injector/annotations"
+	"github.com/dapr/dapr/pkg/injector/sidecar"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	componentsUnixDomainSocketVolume      = "dapr-components-unix-domain-socket" // Name of the Unix domain socket volume for components.
+	componentsUnixDomainSocketMountEnvVar = "DAPR_COMPONENT_SOCKET_FOLDER"
```

Comment:
Here too I think we should use a name consistent with the annotation

Reply:
I think the env var is ok, right? I changed to `socket-folder` the annotation

Reply:
Now note that the annotation is "sockets-folder" but the env var is "SOCKET_FOLDER".

Reply:
Yes, that's correct. Because the env var is assigned to each pluggable component container "SOCKET_FOLDER" but the annotation is for all pluggable components from Dapr point of view (but actually nothing prevents to create more than one socket per pluggable component - which is by design)